tests/operators/_asynchronous/test_bigquery.py

```diff
@@ -134,15 +132,15 @@ def test_store_compiled_sql(mock_rendered_ti, mock_get_remote_sql, profile_confi
     mock_task_instance.task = operator
     mock_context = {"ti": mock_task_instance}
 
-    operator._store_compiled_sql(mock_context, session=mock_session)
+    operator._store_template_fields(mock_context, session=mock_session)
 
```

Comment:
@joppevos would it make sense to add some checks here for `operator.gcp_project` and `operator.dataset`?
Happy for us to merge after this. We'll release it as part of Cosmos 1.9.1